mammoth/utils/report_manager.py

```diff
@@ -51,6 +51,10 @@ def start(self):
     def log(self, *args, **kwargs):
         logger.info(*args, **kwargs)
 
+    def log_valid(self, *args, **kwargs):
+        if valid_logger is not None:
+            valid_logger.info(*args, **kwargs)
```

Comment:
I would have a backup default of logging to the general logger when the valid logger is undefined